apps/jobs/migrations/0024_submission_static_code_upload_submission_log_file.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.20 on 2022-10-25 21:22
+
+import base.utils
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobs', '0023_submission_enverr_file'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='static_code_upload_environment_log_file',
+            field=models.FileField(blank=True, null=True, upload_to=base.utils.RandomFileName('submission_files/submission_{id}')),
+        ),
+    ]
```

Comment:
Please delete these migration files and regenerate them

Reply:
I regenerated them, but it caused numerous issues. Should I just delete this and subsequent apps/jobs migrations?